jax/_src/scipy/sparse/linalg.py

```diff
@@ -483,13 +484,13 @@ def body_fun(value):
 
   initialization = (x0, 0, unit_residual, residual_norm)
   x_final, k, _, err = lax.while_loop(cond_fun, body_fun, initialization)
-  _ = k # Until we can pass this out
+  _ = k  # Until we can pass this out
```

Comment:
This is completely tangental to this PR but is there an open issue for tracking this missing feature?

Reply:
Yes, see https://github.com/google/jax/issues/4322

Reply:
Great, thanks for the pointer!